### ISSUE TITLE
There is an error in extracting the vocabulary file and the image data

### ISSUE THREAD

**Issue by yls-nwu** [SoftWiser-group/MaCon#1]
![Uploading image.png…]()
There is an error in extracting the vocabulary file and the image data.Can you upload it again.
Also when I upload the image data to the server, the image data decompression will report an error,do you 
encounter this situation.

Thank you very much for your reply

**Comment by Zhangtd** [comment]
No, I never encounter this problem before. Are the zip files broken?

**Comment by InTheFuture7** [comment]
@yls-nwu Hi, have you downloaded the **Instagram.data or InstagramImage.zip files in data/README.txt** before the links expired? If so, would you be willing to share them? Thanks!
